scripts/tests/test_convert_tensors.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import tempfile
+
+from scilpy.io.fetcher import fetch_data, get_home, get_testing_files_dict
+
+# If they already exist, this only takes 5 seconds (check md5sum)
+fetch_data(get_testing_files_dict(), keys=['processing.zip'])
+tmp_dir = tempfile.TemporaryDirectory()
+
+
+def test_help_option(script_runner):
+    ret = script_runner.run('scil_convert_tensors.py', '--help')
+    assert ret.success
+
+
+def test_execution_processing(script_runner):
+    os.chdir(os.path.expanduser(tmp_dir.name))
+
+    # No tensor in the current test data! I'm running the compute_dti_metrics
```

Comment:
Does it take long to compute ? To shorten the running time of tests, would it be complicated to add tensors in the test data ? @arnaudbore